improver_tests/acceptance/SHA256SUMS

```diff
@@ -994,8 +994,9 @@ ea67ae7a7f5363ae3692b29c93fb9989449d96e49dc613a1297d98ddb12c578a  ./weather-symb
 51f636314a6d1fa894ab98cf750493503b191a779c67d6a15081aab2a3612a31  ./weather-symbol-modes/spot_ties/20201209T1700Z-weather_symbols-PT01H.nc
 134fb1750cf47e868ee67801b1ea5b1f17120e6f58a787a69e54638d7d88ff82  ./weather-symbol-modes/spot_ties/20201209T1800Z-weather_symbols-PT01H.nc
 391a9bd50824318fb8f147db3b14bcd56f5b0f0e0d5479b3579e08d70ef6ba77  ./weather-symbol-modes/spot_ties/kgo.nc
+391a9bd50824318fb8f147db3b14bcd56f5b0f0e0d5479b3579e08d70ef6ba77  ./weather-symbol-modes/spot_ties/new_kgo.nc
```

Comment:
Is this new line right? I can't see a "new_kgo" in the original and is it supposed to have the same checksum as the one above?